### DISCUSSION TITLE
New release...with SD3 support

### DISCUSSION THREAD

**Discussion by vladmandic** [vladmandic/sdnext#3222] [Announcements]
## See full announcement for details
<https://github.com/vladmandic/automatic/discussions/99#discussioncomment-9764747>

**Comment by tritant** [comment]
Hello, how tu use SD3? when y try to load model y have this issue
Edit: resolved with token acces.
![image](https://github.com/vladmandic/automatic/assets/15909062/839cff62-c9c5-4232-8146-c0769f7427ae)

**Reply by brknsoul** [reply]
Go to the website in the first link, enter info (doesn't have to be real), tick the box, click Accept. Restart SD.Next.

**Comment by Vasuda** [comment]
This is really bad implementation. Why does it insist on communicating with hug in the first place? I already got all the resources downloaded to the relevant dirs before for other uis, even if this crappy autodownload worked  - and it doesnt, if you need to sign up, its not automatic, just manual with extra steps - it would still be undesirable to redownload the same stuff for no reason. 

And on top of that its very unclear how to resolve this error too. Going to the links, accessing the repo etc. etc. doesnt do anything.

**Reply by vladmandic** [reply]
entitled much? sorry, you don't like it, don't use it. this is free and open source.
and you need to sign up because that is how stabilityai set it up, not sdnext. and sdnext was released before sd3 was available on other platforms such as civitai.

**Reply by Vasuda** [reply]
Lol. Insecure much? My whole point was that the functionality would've been better if you had done less, not more. It doesnt matter how SAI is setup, the problem is that its being accessed at all, instead of looking into the local folder that all other models are fetched from. 

But i guess cant expect much if basic feedback is received with such a hissy fit... And yea, i'll just continue not using it then.

**Reply by brknsoul** [reply]
SD3 was implemented in SD.Next when HF was the only source for SD3.

Chill, dude.

**Comment by bluntstone** [comment]
Took a few minutes to figure out. I agree just download the model and go.
That's what I did in comfyui. Sure future updates will take care of it.

**Comment by vladmandic** [comment]
create issue, don't comment on announcement threads!
